src/components/VFilters/VSearchGridFilter.vue

```diff
@@ -115,13 +116,14 @@ export default defineComponent({
      * This watcher fires even when the queries are equal. We update the path only
      * when the queries change.
      */
-    watch(
+    watchDebounced(
       () => searchStore.searchQueryParams,
       (newQuery: ApiQueryParams, oldQuery: ApiQueryParams) => {
         if (!areQueriesEqual(newQuery, oldQuery)) {
           router.push(searchStore.getSearchPath())
         }
-      }
+      },
+      { debounce: 800, maxWait: 5000 }
```

Comment:
What is the reasoning behind these values?

Reply:
Ah I see @zackkrida wanted to raise it from 500. 800 seemed a very specific, very arbitrary choice to start with.